new_lineage_panel/src/App.tsx

```diff
@@ -339,6 +340,7 @@ function App() {
               >
                 <Background />
                 <Controls />
+                <LineageLegend />
```

Comment:
i think we should move `LineageLegend` out of `ReactFlow` tag. cant say how well this will behave with `ReactFlow` canvas especially with for z-index.

Reply:
fixed